docs/generic-templates.md

```diff
@@ -0,0 +1,173 @@
+# Generic templates
+
+This presentation ingores a few things which are easy to implement but would add unnecessary complexity to the presentation:
```

Comment:
”ingores”

Reply:
You'd think @dajmaki might be a bit more flexbile with the spelling eh @martin-drhu-da? :)